gyrinx/core/templates/core/crew/crew_form.html

```
@@ -17,60 +17,90 @@ <h1 class="h3">
         </h1>
-        <p class="text-secondary">
-            Choose this crew's fighters for {{ battle.name }}. Tick the ones you're
-            hand-picking; the dice draw <em>extra</em> fighters at random when the crew is
-            locked at battle start.
-        </p>
+        <p class="text-secondary mb-0">Choose this crew's fighters for {{ battle.name }}, the way the scenario says.</p>
+        <div>
+            <div class="form-label mb-1">Selection method</div>
+            <div class="btn-group" role="group" aria-label="Selection method">
+                {% for entry in method_picker %}
+                    <a href="{{ entry.url }}"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)